backend/index.js

```diff
@@ -10,6 +10,22 @@ app.get('/',(req,res)=>{
     res.send('Hello world');
 })
 
+app.get('/yelp',(req,res)=>{
```

Comment:
Can we name the endpoint something a little bit more descriptive. If I knew nothing about this api other than the name of it, would '/yelp' tell me what this api is doing?

Reply:
I changed it to /yelpGetThingsToDo, is that too long or anything. Is there a certain format I should follow?